src/converter/group.py

```diff
@@ -18,7 +18,7 @@ def convert(fig_group):
 def post_process_frame(fig_group: dict, sketch_group: Group) -> Group:
     convert_frame_style(fig_group, sketch_group)
 
-    if fig_group.get("resizeToFit", False):
+    if fig_group.get("resizeToFit", True):
```

Comment:
So in the `fig` model a `frame` with `resizeToFit` set to `true` seems to be what makes it a "group" rather than a "frame. 

Flipping this switch seems to make the resizing constraints behave correctly, but again, this will need more testing prior to merge.